src/ontology/rejected-mappings.sssom.tsv

```diff
@@ -0,0 +1,26 @@
+subject_id	subject_label	object_id	predicate_id	object_label	mapping_justification	mapping_tool	confidence	subject_match_field	object_match_field	match_string	comment	predicate_modifier	reviewer_id	reason for rejected
+MONDO:0005571	polycythemia	NCIT:C27794	MONDO:equivalentTo	Polycythemia (Excluding Polycythemia Vera)	semapv:LexicalMatching	oaklib	0.8497788952	rdfs:label	rdfs:label	polycythemia	LEXMATCH	NOT	https://orcid.org/0000-0002-4142-7153	mapping rule incorrect

```

Comment:
Invalid `predicate_id`, see #333; having this correct is important for the code here to work, as we only reject mappings with the exact same predicate_id!